RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1429,6 +1429,23 @@ boolean auto_knapsackAutoConsume(string type, boolean simulate)
 	return true;
 }
 
+int auto_spleenFamiliarAdvItemsPossessed() 
+{
+	//returns how many size 4 items from spleen familiars in possession
+	
+	int spleenFamiliarAdvItemsCount = 0;
+	
+	foreach it in $items[Unconscious Collective Dream Jar, Grim Fairy Tale, Powdered Gold, Groose Grease, beastly paste, bug paste, cosmic paste, oily paste, demonic paste, gooey paste, elemental paste, Crimbo paste, fishy paste, goblin paste, hippy paste, hobo paste, indescribably horrible paste, greasy paste, Mer-kin paste, orc paste, penguin paste, pirate paste, chlorophyll paste, slimy paste, ectoplasmic paste, strange paste, Agua De Vida]
+	{
+		if(auto_is_valid(it))
```

Comment:
I am thinking maybe
`if(auto_is_valid(it) && mall_price(it) < 20000)`
to avoid consuming excessively valuable pastes. thoughts?
@Malibu-Stacey what do you think?

Reply:
don't hardcode values like 20000. Use get_property("autoBuyPriceLimit").to_int() but otherwise I agree.

Reply:
```suggestion
		if(auto_is_valid(it) && mall_price(it) < get_property("autoBuyPriceLimit").to_int())
```